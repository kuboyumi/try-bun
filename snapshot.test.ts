import { test, expect } from 'bun:test';

test('snapshot', () => {
  const obj = { a: 1, b: 2 };
  expect(obj).toMatchSnapshot();
});
