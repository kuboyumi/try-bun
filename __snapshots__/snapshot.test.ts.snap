// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
{
  "a": 1,
  "b": 2,
}
`;
